edge/pkg/eventbus/common/util/common.go

```diff
@@ -50,8 +53,39 @@ func HubClientInit(server, clientID, username, password string) *MQTT.ClientOpti
 			opts.SetPassword(password)
 		}
 	}
-	tlsConfig := &tls.Config{InsecureSkipVerify: true, ClientAuth: tls.NoClientCert}
+
+	klog.Infof("Start to config MQTT client tls connect")
+	tlsConfig := &tls.Config{}
+	if eventconfig.Config.Tls.Enable == true {
```

Comment:
```suggestion
	if eventconfig.Config.Tls.Enable {
```

Reply:
thanks for your comments, now these problems has been modified